### ISSUE TITLE
[FEATURE] [WASM] Expose canvasSize in SkiaSharp.Views.Blazor

### ISSUE THREAD

**Issue by JensKrumsieck** [mono/SkiaSharp#1911]
**Is your feature request related to a problem? Please describe.**
analogous to https://github.com/mono/SkiaSharp/issues/1831 & https://github.com/mono/SkiaSharp/pull/1832 it may is useful o expose the canvasSize Property, too. As suggested here: https://github.com/mono/SkiaSharp/discussions/1888

**Describe the solution you'd like**
Add a public property on both components to expose the private field.

**Describe alternatives you've considered**

--
**Additional context**

--

**Comment by simonsarris** [comment]
Yeah, anyone who wants to do non-trivial JS interop (in addition to merely drawing) with this `<canvas>` element needs this exposed.
